backend/requirements/requirements.txt

```diff
@@ -0,0 +1,33 @@
+-r base.txt
+
+Werkzeug[watchdog]==2.2.2 # https://github.com/pallets/werkzeug
+ipdb==0.13.11  # https://github.com/gotcha/ipdb
+watchgod==0.8.2  # https://github.com/samuelcolvin/watchgod
+
+# Testing
+# ------------------------------------------------------------------------------
+mypy==0.991  # https://github.com/python/mypy
+django-stubs==1.13.1  # https://github.com/typeddjango/django-stubs
+pytest==7.2.0  # https://github.com/pytest-dev/pytest
+pytest-sugar==0.9.6  # https://github.com/Frozenball/pytest-sugar
+djangorestframework-stubs==1.8.0  # https://github.com/typeddjango/djangorestframework-stubs
+
+# Code quality
+# ------------------------------------------------------------------------------
+flake8==6.0.0  # https://github.com/PyCQA/flake8
+flake8-isort==6.0.0  # https://github.com/gforcada/flake8-isort
+mypy==0.991
+coverage==7.0.3  # https://github.com/nedbat/coveragepy
+black==22.12.0  # https://github.com/psf/black
+pylint-django==2.5.3  # https://github.com/PyCQA/pylint-django
+pylint-celery==0.3  # https://github.com/PyCQA/pylint-celery
+pre-commit==2.21.0  # https://github.com/pre-commit/pre-commit
+
+# Django
+# ------------------------------------------------------------------------------
+factory-boy==3.2.1  # https://github.com/FactoryBoy/factory_boy
+model-bakery==1.9.0  # https://github.com/model-bakers/model_bakery
+django-debug-toolbar==3.8.1  # https://github.com/jazzband/django-debug-toolbar
+django-extensions==3.2.1  # https://github.com/django-extensions/django-extensions
+django-coverage-plugin==3.0.0  # https://github.com/nedbat/django_coverage_plugin
+pytest-django==4.5.2  # https://github.com/pytest-dev/pytest-django
```

Comment:
Lets just set the deps that we use as we go, and not copy paste

Reply:
k